src/components/Fieldset/DynamicFieldset/DynamicFieldset.types.ts

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+
+type BaseProps = {
+    /**
+     * Dynamic fieldset label
+     */
+    label: string;
+
+    /**
+     * Dynamic fieldset's id
+     */
+    id: string;
+
+    /**
+     * Additional classname
+     */
+    className?: string;
+
+    /**
+     * Dynamic fieldset helper
+     */
+    helper?: string;
+
+    /**
+     * Define fieldset field(s)
+     */
+    children: React.ReactElement;
```

Comment:
I checked, and there are some cases when the fieldset component is used as a boolean, so no children are provided.
So `children` become an optional prop.